soap/src/java/com/zimbra/soap/account/message/GetInfoRequest.java

```diff
@@ -37,6 +41,7 @@
  *     sections you want in the "sections" attr.
  */
 @XmlRootElement(name=AccountConstants.E_GET_INFO_REQUEST)
+@GraphQLType(name=GqlConstants.CLASS_GET_INFO_REQUEST, description="Get information about an account")
```

Comment:
don't need GQL annotations in this class as it's not used in the graphql schema.